crimeApi/index.js

```diff
@@ -0,0 +1,87 @@
+var http = require('http');
+
+//import express
+var express = require('express');
+
+//report Route to send images to the uploads folder
+var reportRoute = require('./route/report');
+
+//Image Route to send images to the uploads folder
+var imageRoute = require('./route/image')
+
+//This is our image file in the route folder
+// var image = require('./route/image'); 
+
+app = express(),
+port = process.env.PORT || 3000,
+bodyParser = require('body-parser');
+
+//File system
+var fs = require("fs");
+
+//We need to connect to our database CrimeMap
+//1.  We need to include our mongodb module so we can us it.
+// 2. We create a server instance for MongoD the same way we did for node js
+// 3. Remember to create a database on MongoDB we called ours CrimeMapp
+//4. We can now connect to our database
+var mongo = require('mongodb'),
+Server = mongo.Server,
+// Db = mongo.Db;
+
+
+//MongoDb has its own port number localhost://27017
+var server = new Server('localhost', 27017, {auto_reconnect: true});
+//var db = new Db('CrimeMap', server);
+
+//Show a message if we connected successfully to our mongodb server
```

Comment:
Why are there so many commented lines? Do you plan to include this at some point?